optuna/pruners/hyperband.py

```diff
@@ -57,19 +57,67 @@ class HyperbandPruner(BasePruner):
         most use cases).　Please see Section 3.6 of the `original paper
         <http://www.jmlr.org/papers/volume18/16-558/16-558.pdf>`_ for the detail.
 
+    Example:
+
+        We minimize an objective function with Hyperband pruning algorithm.
+
+        .. testsetup::
+
+            import numpy as np
+            from sklearn.model_selection import train_test_split
+
+            np.random.seed(seed=0)
+
+        .. testcode::
+
+            import optuna
+            from sklearn.linear_model import SGDClassifier
+
+            X = np.random.randn(200).reshape(-1, 1)
+            y = np.where(X[:, 0] < 0.5, 0, 1)
+            X_train, X_test, y_train, y_test = train_test_split(X, y, random_state=0)
+            classes = np.unique(y)
+            n_train_iter = 100
```

Comment:
I'm suspecting @toshihikoyanase 's last comment was to only expose `n_train_iter`, and not the rest of the setup (e.g. `X`, `y`)?

Reply:
@hvy Thanks! I mentioned only `n_train_iter`.